renderer/viewer/three/world/vrHud.ts

```
@@ -0,0 +1,129 @@
+import * as THREE from 'three'
+import { WorldRendererThree } from '../worldrendererThree'
+
+export class VRHud {
+  private hudMesh: THREE.Mesh
+  private hudCanvas: HTMLCanvasElement
+  private hudContext: CanvasRenderingContext2D
+  private hudTexture: THREE.CanvasTexture
+  private hudGroup: THREE.Group
+  
+  constructor(private worldRenderer: WorldRendererThree) {
```

ESLint

[error] Missing space before function parentheses.

(@stylistic/space-before-function-paren)